docs/user-guide/recipes.rst

```diff
@@ -942,6 +942,118 @@ And now we can see that ``repo_2`` has the category, groups, and RPMs::
     Vendor:
     Version:      0.3.1
 
+
+Package Environments
```

Comment:
I guess I should also write a release note that now it is possible to upload package_environment via CLI and API?